var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -46,7 +46,16 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
 
     conflicts("~hiop~ipopt", msg="ExaGO needs at least one solver enabled")
 
-    # Dependencides
+    # Profling and optimization flags
+    variant(
+        "full_optimizations",
+        default=False,
+        description="Enable/Disable optimizations and release type",
+    )
+    # variant("with_profiling", default="none",
```

Comment:
I'm happy to leave in these comments as an indication that we are working on adding this sort of support, even though it isn't productional yet.